spec/lib/path_string_spec.rb

```diff
@@ -6,7 +6,14 @@
   specify { expect(subject).to eql GrapeSwagger::DocMethods::PathString }
   specify { expect(subject).to respond_to :build }
 
-  describe 'operation_id_object' do
+  describe 'path_string_object' do
+    specify 'The original route path is not mutated' do
+      route = Struct.new(:version, :path).new
+      route.path = '/foo/:dynamic/bar'
+      subject.build(route, add_version: true)
```

Comment:
I would like to see an expectation here as well to see that the `path` was modified, but `route.path` was not.